tests/functional/u/unused/unused_private_member.py

```diff
@@ -135,3 +135,26 @@ def __init__(self):
     def attr_c(self):
         """Get c."""
         return cls.__attr_c  # [undefined-variable]
+
+# Test cases for false-positive reported in #4668
```

Comment:
Can you add an additional empty line here? That will make it easier to distinguish from the other test cases.